build.sbt

```diff
@@ -516,10 +522,10 @@ lazy val binCompatTest = project
         else //We are not testing BC on Scala 2.13 yet.
           "org.typelevel" %% "cats-core" % version.value % Provided
       },
-      "org.typelevel" %% "cats-core" % version.value % Test,
       "org.scalatest" %%% "scalatest" % scalatestVersion(scalaVersion.value) % Test
     )
   )
+  .dependsOn(coreJVM % Test)
```

Comment:
I didn't realize that the approach in #2509 would require a `publishLocal` first. This here may be simpler.

Reply:
right. I was testing against existing jars.